app/adapters/alma_adapter/availability_status.rb

```diff
@@ -159,5 +159,10 @@ def cdl_holding?(holding_id)
         end
         cdl
       end
+
+      # status label including library name and location in library
+      def holding_status_label(holding)
```

Comment:
I see why you chose this name but it makes me think that this method returns some status information. Could we call it `holding_location_label`?

Reply:
yes I will update it. thanks